XamTwitch/XamTwitch/Constants.cs

```diff
@@ -7,5 +7,19 @@ public static class Constants
         public const string TwitchApiUri = "https://api.twitch.tv/";
         public const string TwitchPlaylistApiUri = "https://usher.ttvnw.net/";
         public const string ClientIDHeaderKey = "Client-ID";
+
+        public static string AppName = "XamTwitch";
+
+        // OAuth
+        public static string iOSClientId = "qk1v6l0d2nodm1bkk6sqv6ivueh9j2";
```

Comment:
Was it intentional to store secrets in code?

Reply:
good catch, I should remove them